client/src/components/Upvote/Upvote.js

```diff
@@ -5,7 +5,7 @@ import { Divider } from '@blueprintjs/core';
 
 const Upvote = props => {
   const upvoteComponentClass = props.upvoted ? classes.upvoted : classes.didNotUpvote;
-  const upvoteIcon = props.upvoted ? <UpvoteIcon upvoted /> : <UpvoteIcon />;
+  const upvoteIcon = props.upvoted ? <UpvoteIcon upvoted={true} /> : <UpvoteIcon />;
```

Comment:
test?

Reply:
Tests are added now. Please approve it so that I can merge this PR.